esti/s3_gateway_test.go

```diff
@@ -289,6 +289,88 @@ func TestS3IfNoneMatch(t *testing.T) {
 	}
 }
 
+func TestListMultipartUploads(t *testing.T) {
+	ctx, logger, repo := setupTest(t)
+	defer tearDownTest(repo)
+	s3Endpoint := viper.GetString("s3_endpoint")
+	s3Client := createS3Client(s3Endpoint, t)
+	multipartNumberOfParts := 3
+	multipartPartSize := 5 * 1024 * 1024
+
+	blockStoreType := viper.GetString(ViperBlockstoreType)
+	if blockStoreType != "s3" {
+		t.Skip("Skipping test - blockstore type is not s3")
+	}
```

Comment:
Think you wanted this check to be the first thing - before we create s3 client and etc.